package-lock.json

```diff
@@ -0,0 +1,9966 @@
+{
```

Comment:
We should delete `package-lock.json`, it's not needed when you are using `yarn` (yarn has its own `yarn.lock` lock file, and npm has a different lock file called `package-lock.json`). Yarn is able to install all the same packages as npm (they both pull packages from the same place).

Reply:
So just delete `package-lock.json` and only use `yarn`. `yarn add <my-package>` instead of `npm install <my-package>`